client/app/collections/contacts.coffee

```diff
@@ -17,6 +17,13 @@ module.exports = class ContactCollection extends Backbone.Collection
                     display: "#{contact.get 'name'} &lt;#{email.value}&gt;"
                     toString: -> "#{email.value};#{contact.id}"
 
+            contact.get('cozy').forEach (cozy) ->
+                items.push
+                    id         : contact.id
+                    hasPicture : contact.get 'hasPicture'
+                    display    : "#{contact.get 'name'} &lt;#{cozy.value}&gt;"
+                    toString   : -> "#{cozy.value};#{contact.id}"
+
         return items
```

Comment:
Note for later about the global conception of this part :
We should not deal with display or HTML concern in the model part, this is a view's job. But it's too complicated too change it now. Let's keep it for now. :)